vendor/github.com/weaveworks/common/middleware/instrument.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/felixge/httpsnoop"
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus"
+
```

Comment:
nit: there should be no newline here.

Reply:
@pstibrany that's in weaveworks/common, don't think it's relevant to the PR :)

Reply:
🤦